.eslintrc

```diff
@@ -6,6 +6,7 @@
        "ignoreUrls": true,
        "ignorePattern": "^\\s*var\\s.=\\s*(require\\s*\\()|(/)"
      }],
+     "camelcase": 0,
```

Comment:
Yes. You named the [indexes](https://github.com/strongloop/loopback-connector-couchdb2/blob/feature/indexes/test/autoupdate.test.js#L116) `age_index` etc, hence I disabled the camelcase rule.